epitator/importers/import_geonames.py

```diff
@@ -37,13 +37,36 @@
 ]
 
 
-def read_geonames_csv():
+def windows_max_int():
+    # Find largest integer that does not cause an OverflowError for csv.field_size_limit using "binary search"
+    top = 2**32  # Largest int in Python has 64 bit but largest int in Windows has 32 bit
+    lower = 2**30  # The largest int is between 2^30 and 2^32
+    diff = top - lower
+    while diff > 1:
+        try:
+            mid = int(top - lower / 2)
+            csv.field_size_limit(mid)
+            lower = top
+            top = 2 ** 32
+            diff = abs(top - lower)
+        except OverflowError:
+            top_old = top
+            top = int(top_old / 2)
+            diff = abs(top - lower)
```

Comment:
This could cause the loop to end before a valid parameter for csv.field_size_limit has been found.

Reply:
I am unsure whether my last commit (21st of June) solves this better? The other issues, however, are taken care of.

Reply:
It's a lot better. It's difficult to be sure whether the algorithm is correct without thinking about it a lot. Anything you can do to simplify it or test it would be appreciated.